packages/url-state-provider/src/index.test.ts

```diff
@@ -3,13 +3,15 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/* eslint-disable */
```

Comment:
The same here.  Since we have migrated everything to ts we should then argument why we do so or?